src/ResourceManager/Insights/Commands.Insights/Autoscale/NewAzureRmAutoscaleRuleCommand.cs

```diff
@@ -94,12 +94,6 @@ public class NewAzureRmAutoscaleRuleCommand : AzureRMCmdlet
         public ScaleDirection ScaleActionDirection { get; set; }
 
         /// <summary>
-        /// Gets or sets the scale action type
-        /// </summary>
-        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The scale action type for the setting")]
-        public ScaleType ScaleActionScaleType { get; set; }
```

Comment:
If you add a ValidateSet attribute for the parameter, then this will prevent setting it to other values as well.